src/Reddit/Provider.php

```diff
@@ -73,7 +73,7 @@ protected function mapUserToObject(array $user)
 
         return (new User())->setRaw($user)->map([
             'id'   => $user['id'], 'nickname' => $user['name'],
-            'name' => null, 'email' => null, 'avatar' => $avatar,
+            'name' => empty($user['subreddit']['title']) ? null : $user['subreddit']['title'], 'email' => null, 'avatar' => strtok($user['icon_img'], '?'),
```

Comment:
What is the content of `$user['subreddit']['title']`? Doesn't look like name.

Reply:
![image](https://user-images.githubusercontent.com/26674308/89391245-24906180-d732-11ea-9c86-f7903ddf6fcd.png)
This is display name that user can custom

Reply:
Are you sure that that ends up in the `subreddit` key?
That doesn't seem to make much sense.

Reply:
![image](https://user-images.githubusercontent.com/26674308/89392479-c795ab00-d733-11ea-9db0-157b0dfdfae5.png)
I pretty sure it is